test/blueprints/Do.Test.Blueprints.Service.Application/RestApi/Analyzer/Singleton.generated.cs

```diff
@@ -54,6 +54,16 @@ public void TestException(bool handled)
         target.TestException(handled);
     }
 
+    [HttpPost]
+    [Produces("application/json")]
+    [Route("singleton/test-transaction-nullable")]
+    public async Task TestTransactionNullable([FromBody] Entity entity = default)
```

Comment:
we don't get entities as a json, instead we ask for `entityId` and find it from query context.